packages/lambdas/src/cw-doc-contribution.ts

```diff
@@ -17,12 +17,13 @@ const SIGNED_URL_DURATION_SECONDS = 60;
 
 export const handler = Sentry.AWSLambda.wrapHandler(
   async (event: lambda.APIGatewayRequestAuthorizerEvent) => {
-    const fileName = event.queryStringParameters?.[docContributionFileParam];
+    const fileName = event.queryStringParameters?.[docContributionFileParam] ?? "";
+    const key = fileName.startsWith("/") ? fileName.slice(1) : fileName;
 
     if (fileName) {
-      const url = await s3Utils.s3.getSignedUrl("getObject", {
+      const url = s3Utils.s3.getSignedUrl("getObject", {
```

Comment:
removing the `await` was intentional?

Reply:
yeah, this is synchronous - there are also other places with `async` on `getSignedUrl`, but NABD now IMO